index.html

```diff
@@ -0,0 +1,96 @@
+<html>
+
+    <head>
+        <title>hannah kern</title>
+        <style>
+            p {
+            font-family: sans-serif;
+        }
+        </style>
+    </head>
+
+    <h2>
+        hannah kern
+    </h2>
+
+    <h2>
+        about
+    </h2>
+
+    <h2>
+        projects
+    </h2>
+
+    <h2>
+        contact
+    </h2>
+
+    <h1>
+        hello.
+    </h1>
+
+    <img src="mountain.jpg">
+
+    <h2>
+        my name is hannah. i am passionate about helping people through innovative design. 
+    </h2>
+
+    <img src = "flowers.jpg">
+
+    <p>
+        I’m Hannah, a human centered design and development student at Penn State University. I enjoy designing and creativity. 
+    </p>
+
+    <h3>
+        UXPA
+    </h3>
+
+    <p>
+        UXPA strives to give UX students oportunities to gain expreience outside of the classroom. 
+        As the founding treasurer of UXPA, Penn State Chapter, I have put in a lot of work to get the club started. 
+        Leading meetings, recruiting members, and working to become a recognized club are some of my tasks. 
+        We haven't currently finished any projects, but we are well on our way.
+    </p>
+
+    <h3>
+        Zoom-bot
+    </h3>
+
+    <p>
+        The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+        It should assist users by cleaning the train and giving them information including directions and air quality. 
+        The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. 
+        Subway employees will be using the interface to track trains’ cleanliness and location. The high fidelity wireframes have been developed. 
+    </p>
+
+    <h3>
+        Clinique Virtual Reality
+    </h3>
+
+    <p>
+        Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. 
+        It allows the user to scan their face, view the problem areas, and recommends products and solutions. 
+        I worked on creating high fiedlity wireframes for the application.
+    </p>
+
+    <h4>
+        Let's Chat!
+    </h4>
+
+    <h5>
+        <a href="https://drive.google.com/file/d/1VJn_JMpBiWub_GVBn46kDP1N0Syk42X8/view?usp=sharing">resume</a>
+    </h5>
+
+    <img src = "logotype.svg">
+    
+    <img src = "social-media.svg">
+
+    <img src = "email (1).svg">
```

Comment:
This file should be named better

Reply:
actually, all three icons probably should.

And please place all of your images or icons in an folder labeled `images` or `assests` or something.